.pipelines/templates/.builder-release-template.yaml

```diff
@@ -176,14 +176,14 @@ steps:
       SourceFolder: '$(System.DefaultWorkingDirectory)'
       Contents: 'trivy-report.json'
       TargetFolder: '$(Build.ArtifactStagingDirectory)'
-    displayName: Copy trivy report
+    displayName: Copy trivy rootfs report
   - task: CopyFiles@2
     condition: eq(variables['IS_NOT_1804'], 'true')
     inputs:
       SourceFolder: '$(System.DefaultWorkingDirectory)'
-      Contents: 'trivy-images-table.txt'
```

Comment:
Should we remove the table.txt all together? Wondering if it would be useful in the future if someone is manually looking at a run. Or would they just view their test build scans in kusto? This is assuming that the table.txt would have all the information that is in all of the per-container jsons (correct me if that would not be the case)

Reply:
I see, we would still have an artifact for each container image scan. It's just that each appended table entry would be separated into individual json format. If you think having one table.txt file would be necessary, I can keep that logic on top of jsons.